azure/v11.3.2/release.yaml

```
@@ -0,0 +1,51 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: v11.3.2
+spec:
+  state: active
+  date: 2020-06-12T15:00:00Z
```

```suggestion
  date: 2020-06-11T15:00:00Z
```

Updated release date to today